testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
There is a set of functions testing isPrivate(), isPublic(), hasKeyword(), for a number of types.  I fixed the one for the method level, but I have noticed recently that the ones checking for variable and class access are still checking for the private keyword and expecting that to be correct, when there are other rules that override the existence of the 'private' keyword.